Bottom: 87c1becbbfa32f866c8028d77cfe796f0b50e84e
Top:    9e5c5528f59b29a8ad11d7e8ce0ed52e1c6ab01a
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 16:04:41 +0800

Refresh of vhost_net-passing-raw-xdp-buff

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index d707522..57003c9 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -763,7 +763,6 @@ static void handle_tx_zerocopy(struct vhost_net *net, struct socket *sock)
 			msg.msg_control = NULL;
 			ubufs = NULL;
 		}
-
 		total_len += len;
 		if (tx_can_batch(vq, total_len) &&
 		    likely(!vhost_exceeds_maxpend(net))) {
